Core/include/Acts/TrackFitting/KalmanFitter.hpp

```diff
@@ -105,7 +105,7 @@ struct KalmanFitterOptions {
   /// @param mScattering Whether to include multiple scattering
   /// @param eLoss Whether to include energy loss
   /// @param rFiltering Whether to run filtering in reversed direction as smoothing
-  /// @param rfScaling Scale factor for the covariance matrix before the backward filtering
+  /// @param rfScaling Scale factor for the covariance matrix before the NavigationDirection::Backward filtering
```

Comment:
This is probably an unintended find-replace artifact.

Reply:
oh, yes, you're right Stephen!